C++/Navio/Navio2/RGBled.cpp

```diff
@@ -25,7 +25,7 @@ bool RGBled::initialize() {
         pinB->write(OFF);
     }
     else {
-        fprintf(stderr, "Output enable not set. Are you root?\n");
+        fprintf(stderr, "Initialization failed\n");
```

Comment:
It's better to give more context to an error. Like `%s: pins' initialization failed, __FUNCTION__`